code/style.css

```diff
@@ -1,4 +1,393 @@
+* {
+  box-sizing: border-box;
+}
+
 html {
-  background: #0025ff;
+  font-family: 'Oswald', Helvetica;
```

Comment:
Just a question of curiosity: is there a reason you use font size with percentage instead of px? :)